tools/golang/cmd/encrypt/script.go

```diff
@@ -0,0 +1,102 @@
+/* Copyright 2019 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package main
+
+import (
+	"errors"
+	"flag"
+	tinkpb "github.com/google/tink/proto/tink_go_proto"
+	"github.com/subscriptions-project/swg-js/tools/golang/encryption"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+)
+
+type mapFlags map[string]string
+
+const googleDevPublicKeyUrl string = "https://news.google.com/swg/encryption/keys/dev/tink/public_key"
```

Comment:
nit: googleDevPublicKeyURL

Acronyms in Go are constant case. https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Not changed? Or maybe i'm looking at an old branch.